docs/source/articles/integrate-payment-tenders.html.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Integrate Payment Tenders
+excerpt: TODO
+created_at: 2019/10/10
+---
+
+Integrate Payment Tenders
+======================================================================
+
+Workarea provides two default tender types: store credit and credit card, although the credit card implementation requires customization to communicate with a credit card payment service provider in production.
```

Comment:
Change: although the credit card implementation requires customization to communicate with a credit card payment service provider in production

to 

although the credit card implementation requires customization to communicate with a payment service provider in production

The "credit card" is implied in first part of the sentence.